strings/strings.json

```diff
@@ -7483,5 +7483,9 @@
   "NoValuesFound": {
     "en": "No values found.",
     "fr": "Aucune valeur trouvée."
+  },
+  "ProductRecommandation": {
```

Comment:
The component is to preview results, even if this component is more targeted towards e-commerce, we should not assume that the results are Products